test/server/utils/runOpen.test.js

```diff
@@ -80,6 +80,24 @@ describe('runOpen util', () => {
         `);
       });
     });
+
+    it('on specify URL with page in Google Chrome ', () => {
```

Comment:
I think that description doesn't make sense when running in the test runner

Reply:
I did specify http/https in the description, and I added http test, Thanks.

Reply:
👍